rpc/client.go

```diff
@@ -16,12 +21,19 @@ import (
 )
 
 // UploadReportOptions contains all the required options for uploading a report
-// to both Cedar and the Data Pipes service.
+// to both Cedar and SPS.
+
 type UploadReportOptions struct {
 	Report          *poplar.Report
-	ClientConn      *grpc.ClientConn
+	CedarClientConn *grpc.ClientConn
+	SPSClientConn   *http.Client
+	SPSUrl          string
 	SerializeUpload bool
 	DryRun          bool
+
+	SendToCedar  bool //Whether to send the report to Cedar.
+	SendToSPS    bool //Whether to send the report to SPS.
```

Comment:
Is this needed? Why not just use `SendRatioSPS`? If set to 0, that's equivalent to setting this to `false`.

Reply:
I was thinking it would be nice to be explicit, but you're right, that's unnecessary. Removed it